pages/verify/[discordServerId]/[discordMemberId]/[customLink].tsx

```diff
@@ -18,6 +23,27 @@ import styles from "../../../../styles/Verify.module.scss";
 type Props = {
   discordServerName: string;
   starknetNetwork: NetworkName;
+  signMessage: (message: any) => Promise<Signature>;
+};
+
+type StarknetAccount = {
```

Comment:
Please apply this comment https://github.com/nmalzieu/starky/pull/77/files#r1937105706